Runtime/Dispatcher/Dispatcher.cs

```diff
@@ -42,7 +49,7 @@ private void Enqueue(List<Event> data)
                 _events.Enqueue(eventData);
         }
 
-        public async void Flush()
+        public async Task Flush()
         {
```

Comment:
I'm concerned we don't have any kind of thread safety mechanism on the Flush method